pkg/runner/builtin/runner.go

```diff
@@ -283,11 +235,20 @@ func cname(job *types.Job) string {
 
 func (r *Runner) Poll(q queue.Queue) {
 	for {
```

Comment:
yes, I not thinked about this yet. It's like a hack for now. Is there is `retry` for Go?